src/Builder/FrontendDialectHelper.cpp

```diff
@@ -269,10 +269,11 @@ mlir::Type convertONNXTypeToMLIRType(
   case onnx::TensorProto_DataType::TensorProto_DataType_COMPLEX64:
   case onnx::TensorProto_DataType::TensorProto_DataType_COMPLEX128:
   case onnx::TensorProto_DataType::TensorProto_DataType_UNDEFINED:
-  default:
     assert(false && "Unsupported data type encountered.");
```

Comment:
This one could also be turned to an llvm unreachable, just in case you have to re-update the PR